js/functions.js

```diff
@@ -136,6 +136,13 @@
         $mainMenu.removeClass( 'open' );
     });
 
+    $( document ).on( 'click', '#main-menu li', function() {
+        if( window.location.hash ) {
```

Comment:
Just tested with plain URL and `#` at ending. everything is ok. However I what if I use a wrong URL or id based navigation?